### ISSUE TITLE
NS1001 is not reported for ReceivedExtensions.Received calls

### ISSUE THREAD

**Issue by tpodolak** [nsubstitute/NSubstitute.Analyzers#130]
Repro
```c#
using NSubstitute;
using NSubstitute.ReceivedExtensions;

namespace MyNamespace
{
    public class FooTests
    {
        public class Foo
        {
            public int Id { get; }
        }
        
        public void Test()
        {
            var secondSubstitute = Substitute.For<Foo>();
            
            var id = secondSubstitute.Received(Quantity.AtLeastOne()).Id;
        }
    } 
}
```

**Comment by tpodolak** [comment]
Closed by #132
